src/docs/GetStarted/Developers/AdeoPreset/index.mdx

```diff
@@ -33,7 +33,7 @@ module.exports = {
 
 Using the ADEO preset, Mozaic will generate a series of tokens for you to use in your project.
 
-In order to make it easier for you to see and use these tokens, you must define your local token export path. To do this, you must add the following code to the `mozaic.config.js` file, in addition to the elements already included:
+To make it easier for you to see and use these tokens, you must define your local token export path. To do this, you must add the following code to the `mozaic.config.js` file, in addition to the elements already included:
```

Comment:
You must define your local token export path to make it easier for you to see and use these tokens. In order to do that, you must add the following code to the `mozaic.config.js` file in addition to the elements that are already included: